### ISSUE TITLE
Add Intake Commands

### ISSUE THREAD

**Issue by UserC2** [frc2609/Crescendo-2024#31]
**How does this improve the robot or repo?**
* Required to manipulate notes.

**Feature Description**
* Command to intake note (run until intake sensor detects note)
* Command to expel note (reverse intake) for trap and amp
* Command to feed note to shooter (run intake until sensor no longer detects note or for certain period of time)

**Potential Implementation**
All commands _should_ check whether elevator is in the correct position (within a reasonable tolerance) and delay running until it is, _however_ this may not actually be necessary, because this would only have an effect in two situations:
* Elevator lowering after scoring amp or trap
* Elevator raising to amp or trap
And in these two situations, there is no issue with running the intake early:
* If the elevator was lowering after scoring the amp or trap, there is no game piece in the intake so running it does nothing
* If the elevator was raising to the amp or trap, then the intake command can be scheduled to run after the elevator command completes (instead of scheduling them both simultaneously.

TL;DR don't check for the correct position, it's not necessary.

**Comment by UserC2** [comment]
Supposed to be closed by #43.
